src/index.html

```diff
@@ -1,13 +1,50 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page__body">
+    <div class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="product image"
+      class="card__img">
```

Comment:
In a multiline tag, the attributes should always be an indentation level ahead of the tag and it should be closed on a new line, as in the following example:
```html
<input
  type="text"
  name="surname"
  id="surname"
  required
>
```